docs/models.rst

```diff
@@ -86,6 +86,54 @@ The default parameters are optimized for SO-SAT observations. Meaning that the a
 #. spectral tilt from Planck IX 2018, alpha = -0.42
 #. spectral index and temperature from Planck IX 2018, beta = 1.53, T=19.6 K
 
+
+COLines
+=======
+
+This class implements simulations for Galactic CO emission involving the first 3 CO rotational lines, i.e. :math:`J=1-0,2-1,3-2` whose center frequency is respectively at :math:`\nu_0 = 115.3, 230.5,345.8` GHz. The CO emission map templates are the CO Planck maps obtained with ``MILCA`` component separation algorithm (See `Planck paper <https://www.aanda.org/articles/aa/abs/2014/11/aa21553-13/aa21553-13.html>`). The CO maps have been released at the nominal resolution (10 and 5 arcminutes) and they are convolved with a 1 deg gaussian beam. 
```

Comment:
@giuspugl you convolved with the 1 deg beam? why?

Reply:
Planck maps at nominal resolution are very noisy especially at intermediate galactic latitudes and we don't want to inject extra noise level with the templates.  if you remember we agreed in smoothing all the template maps  to 1 deg resolution since we are interested in the diffuse Galactic emission.

Reply:
ok, can you add that to the docs?